activation/nipost.go

```diff
@@ -568,6 +571,8 @@ func (nb *NIPostBuilder) getBestProof(
 			proofs <- &poetProof{
 				poet:       proof,
 				membership: membership,
+				client:     client,
```

Comment:
Instead of passing around the client here, I think it would be better to just pass data and calculate the ticks here rather than later:

```suggestion
				ticks:     proof.LeafCount/client.GetTickSize(),
```

wdyt?

Reply:
`client` is used later also for getting `URL` (from `.Address()` in line 609) to show from which poet instance proof was taken.
Also ticks calculation is not needed for all proofs. Interesting one is calculated only for best one, so we have there optimization as only one division is made instead of `n`, where `n` is the pool size of all proofs ;)